### PR TITLE
Created HtmlToImageRenderer class using swing, graphics, and JEditorP…

### PR DESCRIPTION
**Pull request by MC-Student**
…ane. Public method creates a png image from given html string. Current test passes. Not sure if background is transparent.

Used https://stackoverflow.com/questions/59160419/java-convert-html-to-image

### CODE REVIEW

**Comment by MC-Student** [comment]
Used https://stackoverflow.com/questions/59160419/java-convert-html-to-image

File: src/test/java/touroll/images/HtmlToImageTest.java
Comment (lkenigsb) [inline src/test/java/touroll/images/HtmlToImageTest.java:1]: //given
//when
//then

File: src/main/java/touroll/markdown/HtmlToImageRenderer.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    private final int height;
+    private final BufferedImage image;
+    private final Graphics graphics;
+    private final JEditorPane jep;
+
+    public HtmlToImageRenderer() throws IOException
+    {
+        width = 600;
+        height = 600;
+        image = new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB);
```
Comment (andrewoid) [inline src/main/java/touroll/markdown/HtmlToImageRenderer.java:21]: Try to figure out if you can output an image with a transparent background.

File: src/main/java/touroll/markdown/MarkdownRenderer.java [hunk 2]
```diff
@@ -24,9 +24,9 @@ public String getHtmlRendering(String toRender)
         return renderer.render(document);
     }
 
-    public void writeHtmlStringIntoNewHtmlFile(String rendered) throws IOException
+    public Image getImageFromMarkdown(String toRender) throws IOException
     {
-        File newHtmlFile = new File("rendered.html");
-        FileUtils.writeStringToFile(newHtmlFile, rendered);
+        HtmlToImageRenderer htmlToImageRenderer = new HtmlToImageRenderer();
```
Comment (andrewoid) [inline src/main/java/touroll/markdown/MarkdownRenderer.java]: Only create `HtmlToImageRenderer` once

File: src/main/java/touroll/markdown/HtmlToImageRenderer.java (hunk 1 above)
Comment (MC-Student) [reply src/main/java/touroll/markdown/HtmlToImageRenderer.java:21]: @andrewoid please see latest commit

**Review by andrewoid — APPROVED** [review]

**Pull request merged**